feature/amazon/src/test/java/com/revenuecat/purchases/amazon/AmazonBackendTest.kt

```diff
@@ -33,13 +34,7 @@ class AmazonBackendTest {
     }
     private val dispatcher = SyncDispatcher()
 
-    private var backend: Backend = Backend(
-        API_KEY,
-        mockAppConfig,
-        dispatcher,
-        dispatcher,
-        mockClient
-    )
+    private val backendHelper = BackendHelper(API_KEY, dispatcher, mockAppConfig, mockClient)
```

Comment:
Hmm again, this is not a change from previous behavior, but ideally we would create a new `BackendHelper` (and all mocks) on each test.